src/remote.sh

```diff
@@ -257,3 +258,63 @@ function get_remote_info()
   echo "$ip"
   return 0
 }
+
+# This function manages ssh operations. Currently, we have three basic actions:
+# connect to the VM, execute a command in the VM, and perform a local script
+# inside the VM.
+#
+# @opts Expects a parameter to be passed for the ssh command (it could be
+#       --command, -c, --script, -s). If this parameter receives a null value
+#       this function will make a simple ssh operation; otherwise, it will
+#       attempt to execute a command or script.
+function vm_ssh
```

Comment:
Create another commit just for refactoring this function. For example, `vm_ssh` does not make sense here; we could rename it to `kw_ssh` or something better.